tests/integration/generate_fixtures/common.py

```diff
@@ -210,7 +211,8 @@ def get_process(run_command):
 
 
 def mine_block(w3):
-    origin_block_number = w3.eth.block_number
+    with pytest.warns(DeprecationWarning):
```

Comment:
Do we need this here? It seems like this is just used for the fixture generation so you'd only see it if you were generating the fixture?